behaviors/wysiwyg.js

```diff
@@ -517,19 +517,27 @@ function initWysiwygBinder(enableKeyboardExtras) {
         observer = this.observer,
         data = observer.value() || '', // don't print 'undefined' if there's no data
         editor = createEditor(el, toolbarButtons),
+        boldExtension = _.find(editor.extensions, findExtension('bold')),
         italicExtension = _.find(editor.extensions, findExtension('italic')),
         strikethoughExtension = _.find(editor.extensions, findExtension('strikethrough')),
-        linkExtension = _.find(editor.extensions, findExtension('anchor'));
+        linkExtension = _.find(editor.extensions, findExtension('anchor')),
+        removeFormattingExtension = _.find(editor.extensions, findExtension('removeFormat'));
 
       // apply custom styling to buttons
+      if (boldExtension) {
+        boldExtension.button.innerHTML = `<img src="${site.get('assetPath')}/media/components/clay-kiln/bold.svg" />`;
+      }
       if (italicExtension) {
-        italicExtension.button.innerHTML = '<em>I</em>';
+        italicExtension.button.innerHTML = `<img src="${site.get('assetPath')}/media/components/clay-kiln/italics.svg" />`;
```

Comment:
Would be nice to de-couple the site and kiln. Could this be an option passed into kiln? Or have default images live in kiln?